conf/documentation.conf

```diff
@@ -1182,6 +1182,13 @@ description=<<EOT
 Use the RADIUS username instead of the TLS certificate common name when doing machine authentication.
 EOT
 
+[advanced.vlan_pool_technique]
+type=toggle
+options=round_robbin|username_hash
```

Comment:
You can connect this field with an admin guide section so that users can read the extended help you did.

Refer to the network.dhcpoption82logger field for an example